cql3/assignment_testable.hh

```diff
@@ -32,7 +32,7 @@ public:
     }
 
     static bool is_exact_match(test_result tr) {
-        return tr != test_result::EXACT_MATCH;
+        return tr == test_result::EXACT_MATCH;
```

Comment:
Wow, what a strange bug you fixed here :-)
Luckily, I checked and no existing code uses this function (that was written 9 years ago, commit 18d92f6cd64!) so it's not necessary to split this fix into a separate patch and open an issue for it, as we normally do.